services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+import datetime
+
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str, date:
+        datetime = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()
```

Comment:
You don't need to save() order if `date` is `None`.

Reply:
move your saving to `if` statement